src/__tests__/mock/mapping/mock-instans-metadata.ts

```diff
@@ -0,0 +1,207 @@
+import {IInstanceMetadataContent, ValueType} from "../../../features/configuration/types/Metadata/IntegrationMetadata";
+
+export const MOCK_INSTANCE_METADATA: IInstanceMetadataContent = {
```

Comment:
la til en ny mock data for metadata for at det skulle være mulig å teste med samling i samling.